doc/source/examples/analysis/alignment_and_rms/aligning_trajectory.ipynb

```diff
@@ -49,7 +49,7 @@
     {
```

Comment:
<p>This is somewhat strange, and is rather inconsistent with everything else, why the change? Is it purely because running the notebooks creates a file? Can we not add it to a gitignore instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>